app/controllers/publishers_controller.rb

```diff
@@ -153,7 +153,10 @@ def home
       uphold_connection.create_uphold_cards if uphold_connection.missing_card?
     end
 
-    flash[:notice] = I18n.t("publishers.home.disabled_payouts") if current_publisher.japanese_locale?(params[:locale])
+    current_publisher.update(bat_points_ever_on: true) if current_publisher.japanese_locale?(params[:locale]) || current_publisher.uphold_connection&.japanese_account?
```

Comment:
I think we should be careful here. I don't think locale is enough to enforce BAT Points everywhere. I would think it should be geo based?

Reply:
I'm discussing with Matt